concordium-sdk/src/main/java/com/concordium/sdk/responses/rewardstatus/RewardsOverview.java

```diff
@@ -0,0 +1,78 @@
+package com.concordium.sdk.responses.rewardstatus;
+
+import com.concordium.sdk.responses.ProtocolVersion;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+
+import java.util.Date;
+
+/**
+ * Parsed Rewards Overview from response of
+ * {@link concordium.P2PGrpc.P2PBlockingStub#getRewardStatus(ConcordiumP2PRpc.BlockHash)}
+ */
+@Data
+@Getter
+public class RewardsOverview {
```

Comment:
Please make fields `final` if you use `@Data`. Also then `@Getter` shouldn't be necessary.

Reply:
done